docs/modules/stackablectl/pages/installation.adoc

```diff
@@ -77,7 +77,7 @@ NOTE: If you have a directory in your `$HOME` for user binaries, you can remove
 
 [source,console]
 ----
-$ sudo install -m 755 -t /usr/local/bin stackablectl
+$ sudo install -m 755 /usr/local/bin stackablectl
```

Comment:
question: Is this actually correct? The issue this fixes states `sudo install -m 755 stackablectl /usr/local/bin`.

I sadly have no way to confirm or deny this.

Reply:
Yeah. That's me not reading properly. Fixed: https://github.com/stackabletech/stackable-cockpit/pull/343/commits/7df3edbcc6bdee4fd9153d2dc6b32b4d7e67dfca